### PR TITLE
updates new cohort form styling, adds links to new cohort

### PR DESCRIPTION
**Pull request by nolds9**

### CODE REVIEW

File: app/views/cohorts/index.html.erb [hunk 1]
```diff
@@ -30,4 +30,4 @@
 
 <br>
 
-<%= link_to 'New Cohort', new_cohort_path %>
+<%= button_to 'New Cohort', new_cohort_path %>
```
Comment (jshawl) [inline app/views/cohorts/index.html.erb]: is this for styling purposes?

**Comment by andrewsunglaekim** [comment]
:+1:

File: app/views/cohorts/_form.html.erb [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
   <div class="form-inputs">
-    <%= f.input :name %>
-    <%= f.association :location %>
-    <%= f.association :course %>
-    <%= f.input :start_date %>
-    <%= f.input :end_date %>
+    <%= f.input :name, label: false, placeholder: "Name"  %>
+    <%= f.association :location, label: false, include_blank: "Location"  %>
+    <%= f.association :course, label: false, include_blank: "Course" %>
+    <%= f.input :start_date, label: false  %>
```
Comment (adambray) [inline app/views/cohorts/_form.html.erb]: any way to put a label on these?

File: app/views/cohorts/index.html.erb (hunk 1 above)
Comment (adambray) [inline app/views/cohorts/index.html.erb]: This doesn't work, as it makes a `post` request, not a `get`

**Comment by adambray** [comment]
Good to go after button is fixed!

**Comment by adambray** [comment]
Fixing button now...

**Comment by adambray** [comment]
Never mind, see you did it already, Nick

**Pull request merged**